pkg/porter/explain.go

```diff
@@ -326,6 +326,7 @@ func (p *Porter) printBundleExplainTable(bun *PrintableBundle) error {
 	p.printOutputsExplainBlock(bun)
 	p.printActionsExplainBlock(bun)
 	p.printDependenciesExplainBlock(bun)
+	p.printInstallationInstructionBlock(bun, bundleReference)
```

Comment:
This is really minor but would you please move this line under 331 where we print the tools included in the bundle? That way the output looks like this

```
$ porter explain -r somebundle
// a lot of information about that bundle
// the tools in the bundle
// BIG FINISH: How to install the bundle
```